packages/core/src/App/Components/Layout/Footer/toggle-language-settings.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Icon, Modal, Popover, Text } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import 'Sass/app/modules/settings.scss';
+import LanguageSettings from '../../../Containers/SettingsModal/settings-language';
+
+export type TToggleLanguageSettings = {
+    is_settings_visible: boolean;
+    lang: string;
+    toggleSettings: () => void;
+};
```

Comment:
@yauheni-deriv All of the props are coming from stores, So why not just get them from stores directly? 🤔

Reply:
Alright, will change it

Reply:
@yauheni-deriv It's a new component, So let's use `useStore` and not create more work for the quality team to refactor it later 😬